### DISCUSSION TITLE
ganache cli

### DISCUSSION THREAD

**Discussion by akhilparakka** [smartcontractkit/full-blockchain-solidity-course-py#1578] [Q&A] [answered]
When I try to run a local blockchain from the command line, an error occurs
![Screenshot from 2022-07-11 13-12-59](https://user-images.githubusercontent.com/77268333/178214282-ffb0ba5c-df82-43ea-85a9-045c0ef71d86.png)
.

**Comment by cromewar** [comment, marked as answer]
Hello @Akhil-franko 
The latest release from ganache is not called `ganache-cli` but just `ganache`, so first of all try to install that package.
Also which version of node are you using? I highly recommend to use a LTS one (v16) because the latest version 18 has reported problems with web3 related libraries.

**Reply by akhilparakka** [reply]
I am using node version 18.4.0. I will try with version 16.

**Reply by akhilparakka** [reply]
Got it. Used nvm to change the version. worked fine after that.

**Reply by cromewar** [reply]
I'm glad is solved now ;)
